twill-api/src/main/java/org/apache/twill/api/Configs.java

```diff
@@ -75,6 +82,7 @@ private Keys() {
      */
     public static final int JAVA_RESERVED_MEMORY_MB = 200;
 
+    public static final double HEAP_RESERVED_MIN_RATIO_DEFAULT = Constants.HEAP_MIN_RATIO;
```

Comment:
Inside the `Defaults` class, no need to repeat the variable with `DEFAULT`. Just name it the same as the config key constant.